Makefile

```diff
@@ -18,3 +18,101 @@ $(BUILD_TARGETS): go.sum $(BUILDDIR)/
 
 $(BUILDDIR)/:
 	mkdir -p $(BUILDDIR)/
+
+###############################################################################
+###                                Protobuf                                 ###
+###############################################################################
+
+containerProtoVer=v0.2
```

Comment:
```suggestion
containerProtoVer=v0.7
```

Reply:
@marbar3778 Was the `buf` command changed for the new version? 
```
make proto-gen
Generating Protobuf files
unknown command "protoc" for "buf"
```

Reply:
ah yea, I would recommend migrating. here is how we do it in the sdk. https://github.com/cosmos/cosmos-sdk/blob/main/scripts/protocgen.sh. Repos in the cosmos org should migrate to keep it aligned.

Reply:
I don't really have experience with generating protobufs. Seems like `buf generate` requires a template, i.e., `buf.gen.gogo.yaml`. However, when trying to use the one from the sdk, I have an issue with the dependencies since `-I "third_party/proto"` is missing. Also, should we have a `buf.yaml` as in the sdk (https://github.com/cosmos/cosmos-sdk/blob/main/proto/buf.yaml)? 

@sainoe Do you know how to fix this?

Reply:
I can help jump in later this week.